README.md

```diff
@@ -74,8 +74,12 @@ Versions
 --------
 
 ### 1.0.0.0
-* Initial public release of xRobocopy module with following resources:
-	* xRobocopy
+ - Initial public release of xRobocopy module with following resources:
+    - xRobocopy
+ 
+### 1.1.0.0

```

Comment:
this should be added as "Unreleased" section. We will bump the version number when we release to PS Gallery